ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -197,6 +198,7 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: 'Start ${dir1}, launch towards ${dir2}',
           cn: '从 ${dir1} 飞向 ${dir2}',
+          ko: '${dir1}에서 ${dir2}으로',
```

Comment:
![image](https://github.com/user-attachments/assets/68257040-0f2f-4c71-b1a2-f94f679db545)
여기는 그냥 이동하는게 아니라, 진짜로 발사되는거니까
```suggestion
          ko: '${dir1}에서 ${dir2}으로 발사',
```
로 하는게 좋을 것 같습니다.